src/Pact/Gas/Table.hs

```diff
@@ -39,6 +39,9 @@ data GasCostConfig = GasCostConfig
   , _gasCostConfig_sortFactor :: Gas
   , _gasCostConfig_distinctFactor :: Gas
   , _gasCostConfig_concatenationFactor :: Gas
+  , _gasCostConfig_textConcatenationFactorOffset :: Gas -- up-front cost of text concatenation
+  , _gasCostConfig_textConcatenationFactorStringLen :: Gas -- additional cost of concatenation per 1K characters in the average string
+  , _gasCostConfig_textConcatenationFactorStrings :: Gas -- additional cost of concatenation per 1K strings
```

Comment:
These could be in terms of `MilliGas`

Reply:
You are very correct! I just did this with Robert and the tests are passing.